tools/e2e/examples/minikube/kumactl_workflow.sh

```diff
@@ -84,13 +84,13 @@ run kumactl config view
 run kumactl config control-planes list
 
 # Install kubectl
-run curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/${GOOS:-linux}/${GOARCH:-amd64}/kubectl
+run curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/"${GOOS:-linux}"/"${GOARCH:-amd64}"/kubectl
```

Comment:
Looks like this script uses a number of bash-isms, so suggest changing the shell to `#! /usr/bin/env bash`